client/src/partials/cost_center/allocation/allocation.html

```diff
@@ -20,8 +20,8 @@
     <div class="panel-body">
       <form class="form">
         <div class="form-group">
-          <select class="col-xs-2 form-bhima" ng-model="configuration.costCenter" ng-change="performChange()">
-            <option value="{{cc}}" ng-repeat="cc in cost_centers.data">{{cc.text}}</option>
+          <select class="col-xs-2 form-bhima" ng-model="CenterCtrl.configuration.costCenter" ng-change="CenterCtrl.performChange()">
```

Comment:
You should use `ng-options` here.  It's more performant, and allows you to set a default option.  The default option should be "Select a cost center".